### ISSUE TITLE
Unable to install version 2.1.0 from NuGet

### ISSUE THREAD

**Issue by PixelDough** [Martenfur/Nopipeline#10]
After being driven almost to tears by how awful the MonoGame Content Pipeline is, I decided to give this package a whirl. But when I try to install it into my project, I get the error
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3073	The command "dotnet C:\Users\owner\.nuget\packages\nopipeline.task\2.1.0\build\\..\tools\netcoreapp3.1\any\npl.dll """ exited with code -532462766.	PongBreaker	C:\Users\owner\.nuget\packages\nopipeline.task\2.1.0\build\Nopipeline.Task.targets	8	

```
The tutorial on the readme file of this repository says that an NPL Config file is supposed to be created on the first build, but trying to build past this point results in the same error as before. 

Installing the previous version, v2.0.1, does not result in any fatal errors, however, no NPL config file is produced anywhere in the file structure upon building, and instead I get the debug log
```
C:/Users/owner/Documents/MonoGame Projects/PixelDoughEngine/Engine/Content/Content.npl not found!
1>Run with path to .mgcb or .npl config as an argument:
1>    NoPipeline.exe Content/Content.mgcb
1>or
1>    NoPipeline.exe Content/Content.npl
C:/Users/owner/Documents/MonoGame Projects/PongBreaker/PongBreaker/Content/Content.npl not found!
1>Run with path to .mgcb or .npl config as an argument:
1>    NoPipeline.exe Content/Content.mgcb
1>or
1>    NoPipeline.exe Content/Content.npl
```
I have a Content.mgcb in my project, and I have one in the MonoGame Shared Library project in my solution. Could that have anything to do with it?

**Comment by PixelDough** [comment]
So I fiddled around with it a bit more today. I copied the npl file from the example project into my content folders for each project, and once I fixed their directories to match mine, I tried installing the latest version from v2.0.1, and it worked! And it's generating all the proper files on build, as it's supposed to.

It seems like it's not generating an npl file on it's own on install, and the most recent version of Nopipeline won't install AT ALL unless you have the npl files already created.
